packages/rest/src/lib/RequestManager.ts

```diff
@@ -404,14 +406,17 @@ export class RequestManager extends EventEmitter {
 			}
 		}
 
-		const fetchOptions = {
-			agent: this.agent,
-			body: finalBody,
+		const fetchOptions: RequestInit = {
+			body: finalBody!,
 			// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
 			headers: { ...(request.headers ?? {}), ...additionalHeaders, ...headers } as Record<string, string>,
 			method: request.method,
 		};
 
+		if (finalBody !== undefined) {
+			Reflect.set(fetchOptions, 'body', finalBody);
+		}
+
```

Comment:
```suggestion
```

Looks like you forgot to remove this

Reply:
Coincidentally fixed in c6d27d055d82503ecfb4b6282f9cd5166698b98e